plaso/parsers/windefenderhistory.py

```diff
@@ -0,0 +1,429 @@
+# -*- coding: utf-8 -*-
+"""Parser for Windows Defender DetectionHistory files."""
+
+from datetime import datetime, timedelta
+import os
+
+from dtfabric.runtime import data_maps as dtfabric_data_maps
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.containers import time_events
+
+from plaso.lib import definitions
+from plaso.lib import dtfabric_helper
+from plaso.lib import errors
+
+from plaso.parsers import interface
+from plaso.parsers import manager
+
+
+class WinDefenderHistoryEventData(events.EventData):
+  """Windows Defender Detection History event data.
+
+    Attributes:
+        sha256(str): Hash of the file.
```

Comment:
please sort these alphabetically in the docstring and in the init function

Reply:
Done